contributors/guide/bug-bounty.md

```diff
@@ -0,0 +1,67 @@
+# Kubernetes Bug Bounty Program
+
+This program is a **work in progress**. This tracks the currently proposed program, but the bug
+bounty is not currently active.
+
+## Scope
+
+The scope of the Kubernetes project itself is not well defined, which makes defining a bug bounty
+scope difficult. With that in mind, we've attempted to enumerate the featuers, configurations, and
+components that clearly fall inside or outside of our scope. However, this leaves a lot of gray area
+in between. For that, we ask you to use your best judgment, and welcome contributions to help
+clarify the scope.
+
+### In Scope
+
+The following items are explicitly in-scope for the bug bounty program:
+
+**Cluster Attacks:**
+
+- Privilege escalation due to bugs in RBAC, ABAC, pod security policies
+- Authentication bugs due to OIDC, x509 certificate handling,  
+- Privilege escalation through the kubelet APIs
+- Remote code execution in kubelet, api server
+- Arbitrary key injection to etcd via the Kubernetes API
+- Path traversal attacks in API, namespaces, etcd
+- Info leak from publicly accessible unauthenticated endpoints
```

Comment:
clarify that you don't just mean discovering the existence of those endpoints

Reply:
how's this?

>Info leak (e.g. workload names) from publicly accessible unauthenticated endpoints
>  _Excluding interntionally disclosed info, such as Kubernetes version & enabled APIs_